apps/web/src/components/editor/tiptap.tsx

```diff
@@ -242,7 +244,23 @@ function TipTap(props: TipTapProps) {
         const ignoreEdit = transaction.getMeta("ignoreEdit") as boolean;
         if (preventSave || !editor.isEditable || !onChange) return;
 
-        if (!autoSave.current) return;
+        if (!autoSave.current) {
+          setEditorSaveState(id, SaveState.NotSaved);
```

Comment:
We should call something like `onAutoSaveDisabled` or something and let the upper layer handle this.